packages/shortcuts-extension/src/components/ShortcutItem.tsx

```diff
@@ -253,12 +285,14 @@ export class ShortcutItem extends React.Component<
   getResetShortCutLink(): JSX.Element {
```

Comment:
Worth checking if it is safe to rename this public method now. If this PR was targeting 5.0 then sure, otherwise we need to be careful and follow a deprecation cycle (i.e. rename method but keep the old one as a deprecated alias).

Reply:
We should just follow the safe route: keep the old one and deprecate it.

Reply:
Note: we bumped package version for this shortcuts-extension, safe to rename.